testsuite/MDAnalysisTests/analysis/test_leaflet.py

```diff
@@ -27,9 +27,11 @@ class TestLeafletFinder(TestCase):
     def setUp(self):
         self.universe = MDAnalysis.Universe(Martini_membrane_gro, Martini_membrane_gro)
         self.lipid_heads = self.universe.select_atoms("name PO4")
+        self.lipid_head_string = "name PO4"
 
     def tearDown(self):
         del self.universe
+        del self.lipid_head_string
```

Comment:
I wouldn't bother `del`ing a string. We only do this for universes and other big things to help the garbage collector or to help with closing files during test runs where objects seem to be kept alive for the whole test.